src/test/java/de/tum/cit/aet/artemis/exam/ExamIntegrationTest.java

```diff
@@ -392,11 +393,14 @@ void testCreateExam_asInstructor_returnsLocationHeader() throws Exception {
     @Test
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void testCreateExam_asInstructor_returnsBody() throws Exception {
-        Exam exam = ExamFactory.generateExam(course1, "examF");
+        final Exam exam = validExamWithCustomFieldValues();
+        exam.setQuizExamMaxPoints(40);  // this is only returned by the POST endpoint (createExam),
+        // not the GET endpoint (getExam), because it's an unfinished feature
 
-        Exam savedExam = request.postWithResponseBody("/api/exam/courses/" + course1.getId() + "/exams", exam, Exam.class, HttpStatus.CREATED);
+        final Exam savedExam = request.postWithResponseBody("/api/exam/courses/" + course1.getId() + "/exams", exam, Exam.class, HttpStatus.CREATED);
 
-        assertThat(savedExam.getTitle()).isEqualTo(exam.getTitle());
+        checkCustomFieldValuesExamsAreEffectivelyEqual(savedExam, exam);
+        assertThat(savedExam.getQuizExamMaxPoints()).isEqualTo(exam.getQuizExamMaxPoints());
```

Comment:
why is this not encapsulated under `checkCustomFieldValuesExamsAreEffectivelyEqual` as well?

Reply:
explained in the comment where the quiz is assigned 👍